index.ts

```diff
@@ -3,6 +3,6 @@ export { default as FacebookShareButton } from './src/Facebook.astro';
 export { default as HackerNewsShareButton } from './src/HackerNews.astro';
 export { default as LinkedInShareButton } from './src/LinkedIn.astro';
 export { default as RedditShareButton } from './src/Reddit.astro';
-export { default as TwitterShareButton } from './src/Twitter.astro';
```

Comment:
This should be reverted for this release.

Reply:
Hi @silent1mezzo 

Updated pull request